### PR TITLE
BipartiteCheck - Fix for PR #4882

### PR DESCRIPTION
**Pull request by vi-hna-ja**
Fixed requested changes on the PR #4881

### CODE REVIEW

**Comment by vi-hna-ja** [comment]
Kindly review @HarshCasper. Created a new branch to fix the issues requested because couldn't find the previous branch copy on my local machine.

File: Java/graphs/BipartiteCheck.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+
+/* 
+    Sample Input: 
+    4
+    4
+    1 2
+    2 3
+    4 3
+    1 4
```
Comment (ankitaggarwal23) [inline Java/graphs/BipartiteCheck.java]: Use proper display message

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Java/graphs/BipartiteCheck.java (hunk 1 above)
Comment (vi-hna-ja) [reply Java/graphs/BipartiteCheck.java]: Fixed. You can check now.

File: Java/graphs/BipartiteCheck.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+
+/* 
+Sample Input: 
+4
+4
+1 2
+2 3
+4 3
+1 4
```
Comment (ankitaggarwal23) [inline Java/graphs/BipartiteCheck.java]: Use proper display message how you are reading inputs as 4 4 1 2 etc. doesn't make sense what it is use display proper display message in your code.

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by ankitaggarwal23** [comment]
Also resolve merge conflicts.

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by HarshCasper** [comment]
Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Comment by vi-hna-ja** [comment]
Yeah sure!
But I see that there are no Docs-related issues raised in this repository. Can I make the issue on the NeoAlgo-Docs repo?

**Comment by HarshCasper** [comment]
> Yeah sure!
> But I see that there are no Docs-related issues raised in this repository. Can I make the issue on the NeoAlgo-Docs repo?

Yeah. All the issues for the docs will go to that repo

**Pull request merged**